region/tourian/main/Mother Brain Room.json

```diff
@@ -397,32 +397,47 @@
         "canTrickyUseFrozenEnemies",
         "Morph",
         {"or": [
-          "canTrickyJump",
           {"and": [
-            "canDownBack",
+            "Wave",
             {"enemyDamage": {
               "enemy": "Rinka",
               "type": "contact",
               "hits": 1
             }}
+          ]},
+          {"and": [
+            "canTrickyJump",
+            {"or": [
+              {"disableEquipment": "ScrewAttack"},
+              "canDownBack"
+            ]}
```

Comment:
This part applies to the wave strat and should be pulled up a level.

It seems like you are going for `OR:[wave, trickyjump]` +`OR:[Rinka hit, trickyJump]`

Reply:
Good catch, yeah that's what I was going for. Should be fixed now